infra/frontend/service/variables.tf

```diff
@@ -8,9 +8,3 @@ variable "image_tag" {
   description = "image tag to deploy to the environment"
   default     = null
 }
-
-variable "domain" {
-  type        = string
-  description = "DNS domain of the website managed by HHS"
-  default     = null
-}
```

Comment:
this has always been null! isn't never been used! even inside of the PR where it was added!

https://github.com/HHS/simpler-grants-gov/pull/557